### DISCUSSION TITLE
Auto closes

### DISCUSSION THREAD

**Discussion by sana888999** [jaykali/maskphish#20] [Q&A] [answered]
after typing keyworks and entering it closes without showing the link

**Comment by jaykali** [comment]
Hi Sana, Can you please show a screenshot?

**Reply by sana888999** [reply]
it only happens when i use it in windows it works fine in kali but their is another problem it gets detected by google.... its not fud

**Reply by sana888999** [reply]
![image](https://user-images.githubusercontent.com/67665857/109610607-d386f700-7ae1-11eb-830b-855a07d353fc.png)

**Reply by jaykali** [reply]
Some browser shows this. This kind of browser have some extra security to bypass the trick. They just put a warning.

**Reply by jaykali** [reply]
Its not by Google. Some Updated browser tried to fix the issue by giving the warning.

**Reply by sana888999** [reply]
ok no worries thank you

**Comment by jaykali** [comment, marked as answer]
Hope this will help you.

**Comment by ingenium-logic** [comment]
Google job thanks
